docs/05.PORT-API.md

```diff
@@ -151,6 +151,32 @@ jerry_port_get_native_module (jerry_value_t name) /**< module specifier */
 }
 ```
 
+## Promise
+
+```c
+/**
+ * HostPromiseRejectionTracker operations
+ */
+typedef enum
+{
+  JERRY_PROMISE_REJECTION_TRACKER_REJECT, /**< promise is rejected without any handlers */
+  JERRY_PROMISE_REJECTION_TRACKER_HANDLE, /**< handler is added to a rejected promise for the first time */
+} jerry_promise_rejection_tracker_t;
+
+/**
+ * Track unhandled promise rejections.
+ *
+ * Note:
+ *      This port function is called by jerry-core when JERRY_BUILTIN_PROMISE
+ *      is enabled.
+ *
+ * @param promise rejected promise
+ * @param operation HostPromiseRejectionTracker operation
+ */
+void jerry_port_track_promise_rejection (const jerry_value_t promise,
```

Comment:
I was also considering suggesting yet another name change, to have the new function named `jerry_port_promise_rejection_tracker`. That would make the new typedef and the new function names aligned, as well as the name used by the spec. However, existing port API function names usually start with a verb (get this, set that, sleep, print, log, etc. -- the only exception being fatal). So, I'm not sure about this. But we should agree upon a good name so that it doesn't need to be changed, at least in the near future.

Reply:
I'd keep this `jerry_port_track_promise_rejection` for the due to the starting verb convention with the other API functions. 

`jerry_port_promise_rejection_tracker` would be suitable if we would rename `jerry_port_log` to `jerry_port_logger` and so on. So I'd leave as it for now.